packages/neuron-wallet/src/services/ckb-runner.ts

```diff
@@ -130,6 +130,11 @@ export const startCkbNode = async () => {
   })
   currentProcess.stdout?.on('data', data => {
     const dataString: string = data.toString()
+    if (dataString.includes('No space left on device')) {
```

Comment:
is the message thrown from the CKB client? I find another detection in the [CKB](https://github.com/nervosnetwork/ckb/blob/e888ea2f871424de2403f84d25615d676d0719f2/util/app-config/src/sentry_config.rs#L86) repo

Reply:
Of course, I get this log from the ckb run logs.

```
2024-01-14 01:43:41.411 +00:00 GlobalRt-8 ERROR panic  thread 'GlobalRt-8' panicked at 'append block should be OK: DB("IO error: No space left on device: While appending to file: /Volumes/Test_/mainnet/data/indexer/store/004420.log: No space left on device")': util/indexer/src/service.rs:149   0: backtrace::capture::Backtrace::new
   1: ckb_logger_service::setup_panic_logger::{{closure}}
   2: std::panicking::rust_panic_with_hook
   3: std::panicking::begin_panic_handler::{{closure}}
   4: std::sys_common::backtrace::__rust_end_short_backtrace
   5: _rust_begin_unwind
   6: core::panicking::panic_fmt
   7: core::result::unwrap_failed
   8: ckb_indexer::service::IndexerService::try_loop_sync
   9: tokio::runtime::task::raw::poll
  10: std::sys_common::backtrace::__rust_begin_short_backtrace
  11: core::ops::function::FnOnce::call_once{{vtable.shim}}
  12: std::sys::unix::thread::Thread::new::thread_start
  13: __pthread_start

2024-01-14 01:43:41.412 +00:00 ctrl-c INFO ckb_bin::subcommand::run  Trapped exit signal, exiting...
2024-01-14 01:43:41.412 +00:00 GlobalRt-0 INFO ckb_tx_pool::service  TxPool is saving, please wait...
2024-01-14 01:43:41.413 +00:00 GlobalRt-4 INFO ckb_network::network  p2p service event: ListenClose { address: "/ip4/0.0.0.0/tcp/8115" }
2024-01-14 01:43:41.513 +00:00 GlobalRt-0 INFO ckb_tx_pool::process  TxPool save successfully
2024-01-14 01:43:41.692 +00:00 GlobalRt-6 WARN ckb_network::services::dump_peer_store  Dump peer store error, path: "/Volumes/Test_/mainnet/data/network/peer_store" error: Io(Os { code: 28, kind: StorageFull, message: "No space left on device" })
2024-01-14 01:43:44.393 +00:00 main INFO ckb_bin  ckb shutdown
```

Reply:
Okay, the error may be thrown from the rocksdb https://github.com/facebook/rocksdb/blob/21d5a8f54f06e01ca49d4e6bae89bb42ba78dfdd/util/status.cc#L34. We should also handle the same case from CKB, `/No space left/.test(dataString)` would be better